packages/react-native-sdk/src/components/Call/CallControls/ToggleAudioPublishingButton.tsx

```diff
@@ -41,7 +41,7 @@ export const ToggleAudioPublishingButton = ({
     <Restricted requiredGrants={[OwnCapability.SEND_AUDIO]}>
       <CallControlsButton
         onPress={onPress}
-        color={!optimisticIsMute ? colors.static_white : colors.overlay_dark}
+        color={!optimisticIsMute ? colors.static_white : colors.overlay_medium}
```

Comment:
Instead of `overlay_medium` can we make it like `secondaryBackground` ? good to check Figma here, maybe they use a naming scheme